config/local.yaml

```diff
@@ -2,7 +2,7 @@ kind: GenericOperatorConfig
 apiVersion: operator.openshift.io/v1alpha1
 leaderElection:
   disable: true
-interval: "60s"
+interval: "5m"
 storagePath: /tmp/insights-operator
 endpoint: http://[::1]:8081
-#impersonate: system:serviceaccount:openshift-insights:gather
\ No newline at end of file
+impersonate: system:serviceaccount:openshift-insights:gather
```

Comment:
This will make sure we will run under same account locally as we run in production